openwisp_controller/config/migrations/0022_move_to_key_field.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.9 on 2020-01-17 20:32
+
+import re
+
+import django.core.validators
+import openwisp_utils.base
+import openwisp_utils.utils
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('config', '0021_vpn_key'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='OrganizationConfigSettings',
+            name='shared_secret',
+            field=openwisp_utils.base.KeyField(db_index=True, default=openwisp_utils.utils.get_random_key, help_text='used for automatic registration of devices', max_length=32, unique=True, validators=[django.core.validators.RegexValidator(re.compile('^[^\\s/\\.]+$'), code='invalid', message='This value must not contain spaces, dots or slashes.')], verbose_name='shared secret'),
+        ),
+    ]
```

Comment:
same as we discussed in the other PRs, change the old migration and get rid of this one which is unnecessary